apps/test-low-memory.sh

```diff
@@ -1,41 +1,39 @@
 #!/bin/bash
+set -e
 
 # 'npm test' normally does all three of these things.
 # We break them up here so they each run in isolation.
 
-npm run lint && \
-
-node --max_old_space_size=4096 `npm bin`/grunt unitTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF unit; \
-fi && \
-
-node --max_old_space_size=4096 `npm bin`/grunt storybookTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF storybook; \
-fi && \
-
-node --max_old_space_size=4096 `npm bin`/grunt scratchTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF scratch; \
-fi && \
-
-LEVEL_TYPE='turtle' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF integration; \
-fi && \
-
-LEVEL_TYPE='maze|bounce|calc|eval|flappy|studio' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF integration; \
-fi && \
-
-LEVEL_TYPE='applab|gamelab' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF integration; \
-fi && \
-
-LEVEL_TYPE='craft' node --max_old_space_size=4096 `npm bin`/grunt integrationTest && \
-if [ -n "$CIRCLECI" ]; then \
-    bash <(curl -s https://codecov.io/bash) -cF integration; \
+GRUNT_CMD="node --max_old_space_size=4096 `npm bin`/grunt"
+
+if [ -n "$CIRCLECI" ]; then
+  mkdir -p log
+  echo "See apps-test-log under the artifacts tab on circle for test output"
+
+  curl -s https://codecov.io/bash > codecov.sh
+  chmod +x codecov.sh
+
+  $GRUNT_CMD preconcat concat
+
+  SHELL=/bin/bash parallel -j 4 --joblog - ::: "npm run lint" \
```

Comment:
Trying this locally, I get
```
./test-low-memory.sh: line 18: parallel: command not found
```
What provides GNU parallel on Circle?

Reply:
It's in our dockerfile: https://github.com/code-dot-org/code-dot-org/blob/e3e8ffcd7a513f75fd7d6a5db118f5c42e50f5e7/.circleci/Dockerfile#L105